### ISSUE TITLE
"in three weeks" doesn't work

### ISSUE THREAD

**Issue by sils** [scrapinghub/dateparser#291]
```
In [2]: dateparser.parse('in two weeks')
Out[2]: datetime.datetime(2017, 3, 31, 23, 42, 45, 268823)

In [3]: dateparser.parse('in three weeks')

In [4]: dateparser.parse('in 3 weeks')
Out[4]: datetime.datetime(2017, 4, 7, 23, 42, 53, 854388)
```

**Comment by atultherajput** [comment]
Yeah, It is not working for more than two weeks. Examples:

- dateparser.parse('in three weeks')
- dateparser.parse('in five weeks')
- dateparser.parse('in eight weeks')
- dateparser.parse('in eleven weeks')

But these are working:

- dateparser.parse('in one weeks')
- dateparser.parse('in two weeks')
- dateparser.parse('in 3 weeks')
- dateparser.parse('in 5 weeks')
- dateparser.parse('in 8 weeks')
- dateparser.parse('in 11 weeks')

**Comment by atultherajput** [comment]
I think it is because of this:- https://github.com/scrapinghub/dateparser/blob/master/data/languagefiles/en.yaml

`simplifications:
    - day before yesterday: 2 days
    - yesterday: 1 day
    - tomorrow: in 1 day
    - today: 0 days
    - last (week|month|year): 1 \1
    - next (week|month|year): in 1 \1
    - an: 1
    - a: 1
    **- one: 1
    - two: 2**
    - (\d+)\s*hr(s?): \1 hour\2
    - (\d+)\s*min(s?): \1 minute\2
    - (\d+)\s*sec(s?): \1 second\2
    - (?:12\s+)?noon: '12:00'
    - (?:12\s+)?midnight: '00:00'
    - (\d+)h(\d+)m?: \1:\2
    - less than 1 minute ago: 45 seconds
- (?<!from\s)now: 0 second`

**Comment by atultherajput** [comment]
I had submitted a fix for this issue. Now we can parse upto twelve. #299

**Comment by asadurski** [comment]
Hi! Thanks for bringing that up.
The issue is related to https://github.com/scrapinghub/dateparser/issues/46 (and I even have a quite stale branch for that, that I haven't worked on for a while: https://github.com/scrapinghub/dateparser/tree/feature-numerals-to-numbers). It's actually a bit complicated to solve it for more languages than just English. But of course @atultherajput's input is very welcome here for a good start.

**Comment by atultherajput** [comment]
Hey @asadurski 
I would like to work on this issue. I think we can use CLDR database for that.

**Comment by AshwiniPatil5** [comment]
I'm using the pip install to get the dateparser. is this issue fixed in the latest version?

**Comment by Gallaecio** [comment]
Since this specific issue has been fixed, let us keep track of this at #46.
